CSM_GEOMETRY.s

```diff
@@ -111,7 +111,7 @@ CALCSXA		ITA	VLOAD		# PUSHDOWN 00-26D, 28D, 30D, 32D-36D
 # SIGHT LIES ALONG THE STAR VECTOR.
 #
 # THE INPUTS ARE:  1) THE STAR VECTOR REFERRED TO ANY COORDINATE SYSTEM STORED AT STAR.  2) THE NAVIGATION BASE
-# COORDINATES REFERRED TO THE SAME COORDINATE SYSTEM.  THESE THREE HALF-UNIT VECTORS ARE STROED AT XNB, YNB,AND
+# COORDINATES REFERRED TO THE SAME COORDINATE SYSTEM.  THESE THREE HALF-UNIT VECTORS ARE STORED AT XNB, YNB,AND
```

Comment:
Another non-spotted typo!

There's a missing space before `AND` in "STORED AT XNB, YNB,AND" [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0287.jpg)